docs/getting-started/welcome.md

```diff
@@ -4,39 +4,43 @@ sidebar_label: Welcome to UMA
 slug: /
 ---
 
-:::note uUSD has launched
-Learn how to mint and farm `uUSD` tokens with this [tutorial](users/mint-farm-yusd.md).
+## UMA is open-source infrastructure for deploying and enforcing synthetic assets on Ethereum
```

Comment:
Similar to another comment, I think it looks strange to have a title under a title. IMO I think we should create a standard where all pages start with an untitled paragraph